src/Cell.ts

```diff
@@ -2,13 +2,13 @@ import WidgetBase from '@dojo/widget-core/WidgetBase';
 import { RegistryMixin, RegistryMixinProperties }  from '@dojo/widget-core/mixins/Registry';
 import { v } from '@dojo/widget-core/d';
 import { DNode } from '@dojo/widget-core/interfaces';
-import { HasValue, HasColumn, HasItem } from './interfaces';
+import { HasColumn, HasContent, HasItem, HasValue } from './interfaces';
 import { theme, ThemeableMixin, ThemeableProperties } from '@dojo/widget-core/mixins/Themeable';
 
 import * as cellCss from './styles/shared/cell.m.css';
 import * as css from './styles/cell.m.css';
 
-export interface CellProperties extends HasColumn, HasItem, HasValue, RegistryMixinProperties, ThemeableProperties {
+export interface CellProperties extends HasColumn, HasContent, HasItem, HasValue, RegistryMixinProperties, ThemeableProperties {
```

Comment:
What is the purpose of `HasValue` for `Cell`? It's not used in the rendering... is it envisioned that one might want to read the value of `value` from a `Cell` instance?